### PR TITLE
feat(ops): split up E2E testing Cloud Build config

### LINKED ISSUE #756: break up `web-e2e.cloudbuild.yaml` into multiple steps
## Proposal
`web-e2e.cloudbuild.yaml` can be refactored into multiple cloud build steps.

## Problems this will solve
Current e2e test does the following in one single step via bash entrypoint:
- copies files from `client-libs`
- installs python requirements via pip
- runs flask python module
- traverses container filesystem
- installs node dependencies via npm
- runs `npm test`

This makes for a difficult time troubleshooting and reviewing logs.

### PR DESCRIPTION
**Pull request by ace-n**
Fixes #756 

✅ I confirmed that tests pass [here](https://github.com/ace-n/emblem/pull/3).

### CODE REVIEW

File: ops/web-e2e.cloudbuild.yaml [hunk 1]
```diff
@@ -14,6 +14,19 @@
 
 
 steps:
+  - name: bash
```
Comment (rogerthatdev) [inline ops/web-e2e.cloudbuild.yaml:17]: include `id` line to indicate what the step will be doing. Similar to https://github.com/GoogleCloudPlatform/emblem/blob/main/ops/terraform.cloudbuild.yaml 

example

id: copy client library files
... 
id: install python packages

etc

File: ops/web-e2e.cloudbuild.yaml [hunk 2]
```diff
@@ -22,10 +35,8 @@ steps:
     args:
       - '-c'
       - |
-        cp -R ../client-libs/ client-libs
-        pip install -r requirements.txt
         python -m flask run --port ${_FLASK_PORT} &
-        cd ../testing/website-e2e && npm install && npm test
+        cd ../testing/website-e2e && npm test
```
Comment (rogerthatdev) [inline ops/web-e2e.cloudbuild.yaml:39]: this is still using a bash entrypoint to run python and npm in one step. we can split further

File: website/requirements.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
 Flask==2.1.2
 firebase-admin==5.2.0
 gunicorn==20.1.0
+importlib-metadata==5.1.0
```
Comment (rogerthatdev) [inline website/requirements.txt:4]: what's up with this new addition?

**Review by rogerthatdev — COMMENTED** [review]
I tested in my own project and it fails. log attached


[log-bf8ad53a-b621-4bb2-9417-52f2a8e7e511.txt](https://github.com/GoogleCloudPlatform/emblem/files/10365243/log-bf8ad53a-b621-4bb2-9417-52f2a8e7e511.txt)

File: ops/web-e2e.cloudbuild.yaml (hunk 2 above)
Comment (ace-n) [reply ops/web-e2e.cloudbuild.yaml:39]: AFAICT these **must** be run in the same step. (`flask` needs to be running _at the same time_ as `npm test`.)

File: website/requirements.txt (hunk 3 above)
Comment (ace-n) [reply website/requirements.txt:4]: The lack of this module caused a [build failure](https://console.cloud.google.com/cloud-build/builds;region=global/107a22e8-9e46-42f1-966a-5cd8047af6f8?authuser=1&project=anassri-emblem-joonix1):
```
> emblem-e2e-tests@0.0.1 test
> playwright test

Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.8/runpy.py", line 144, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/usr/lib/python3.8/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/workspace/website/flask/__init__.py", line 7, in <module>
    from .app import Flask as Flask
  File "/workspace/website/flask/app.py", line 27, in <module>
    from . import cli
  File "/workspace/website/flask/cli.py", line 39, in <module>
    import importlib_metadata as metadata  # type: ignore
ModuleNotFoundError: No module named 'importlib_metadata'
```

File: ops/web-e2e.cloudbuild.yaml (hunk 1 above)
Comment (ace-n) [reply ops/web-e2e.cloudbuild.yaml]: Done.

**Comment by ace-n** [comment]
@rogerthatdev I think you have a missing Docker image.

Try running the `testing-web-e2e-build-container-trigger` trigger (to build that image) and re-running the tests?

File: ops/web-e2e.cloudbuild.yaml (hunk 2 above)
Comment (rogerthatdev) [reply ops/web-e2e.cloudbuild.yaml:43]: ack

File: website/requirements.txt (hunk 3 above)
Comment (rogerthatdev) [reply website/requirements.txt:4]: ack

**Review by rogerthatdev — COMMENTED** [review]
The e2e test deploys, but when i run it with a change on my Emblem fork, the tests themselves fail
[log-6ac84180-bdce-4745-8d1b-5e980a86b5bc.txt](https://github.com/GoogleCloudPlatform/emblem/files/10365363/log-6ac84180-bdce-4745-8d1b-5e980a86b5bc.txt)

**Review by rogerthatdev — APPROVED** [review]

**Pull request merged**